helm/values.yaml

```diff
@@ -0,0 +1,35 @@
+ingress:
+  enabled: true
+  tls:
+    enabled: true
+  hosts:
+  - localhost
+  path: /
+
+loadBalancer:
+  enabled: true
+
```

Comment:
Perhaps it would be good to put in the comments the options for the `type`, mysql, postgres, and file.

Reply:
You mean inline comments here in the file? I added [this](https://github.com/theupdateframework/notary/pull/1503/files#diff-04c6e90faac2675aa89e2176d2eec7d8R124) on the `README.md`, but I can add that here too if you think that adds value.

Reply:
I think in this case it would be helpful (three options that might take a while for someone to find out). I'm a fan of more commented Helm charts when it's not entirely clear, and I think this Consul chart does a great job: https://github.com/hashicorp/consul-helm/blob/master/values.yaml

Reply:
It definitely makes sense, I'll add that!